lib/resources/r4/related_person_patch.yaml

```diff
@@ -65,6 +65,40 @@ operations:
         <li>If `line` is provided, only the first four values are used. Additional values will be ignored.</li>
       </ul>
 
+  - name: add-identifier
+    path: /identifier/-
+    operation: add
+    type: Identifier
+    url: https://hl7.org/fhir/R4/patient-definitions.html#RelatedPerson.identifier
```

Comment:
This should be related-person-definitions.

Reply:
[fixed](https://github.com/cerner/fhir.cerner.com/pull/555/commits/1530f2ab8911c92413e05c79ec4bf854aa3d1512#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dR72)